### ISSUE TITLE
ProcessCommonJSModules NPE

### ISSUE THREAD

**Issue by anmonteiro** [google/closure-compiler#2345]
I'm getting an `NullPointerException` when trying to process CommonJS modules using the latest February compiler. This doesn't seem like the error reported in #2308.

Below is the relevant part of the stacktrace. I'm still trying to come up with a minimal case, as this happens when trying to consume the `aws-sdk` NPM package (which is huge).
```
Caused by: java.lang.NullPointerException
	at com.google.javascript.jscomp.ProcessCommonJSModules.getScopeInsertionPoint(ProcessCommonJSModules.java:242)
	at com.google.javascript.jscomp.ProcessCommonJSModules.access$400(ProcessCommonJSModules.java:42)
	at com.google.javascript.jscomp.ProcessCommonJSModules$FindImportsAndExports.addGoogProvide(ProcessCommonJSModules.java:556)
	at com.google.javascript.jscomp.ProcessCommonJSModules.process(ProcessCommonJSModules.java:125)
	at com.google.javascript.jscomp.Compiler.processAMDAndCommonJSModules(Compiler.java:1841)
	at com.google.javascript.jscomp.Compiler.parseInputs(Compiler.java:1549)
	at com.google.javascript.jscomp.Compiler.parse(Compiler.java:809)
```

**Comment by ChadKillingsworth** [comment]
A repo case would definitely help.

**Comment by anmonteiro** [comment]
I haven't been able to repro it other than when requiring the `aws-sdk` module, which probably doesn't help narrow it down

**Comment by ChadKillingsworth** [comment]
Closed by e68fcaa
